bin/env_variables_all.txt

```diff
@@ -17,6 +17,9 @@ CCD_BEFTA_MASTER_CASEWORKER_PWD=Pa55word11
 CCD_BEFTA_MASTER_SOLICITOR_1_PWD=Pa55word11
 CCD_BEFTA_MASTER_SOLICITOR_2_PWD=Pa55word11
 CCD_BEFTA_MASTER_SOLICITOR_3_PWD=Pa55word11
+CCD_BEFTA_MASTER_SOLICITOR_4_PWD=Pa55word11
```

Comment:
adding 2 users but 3 passwords ?

Reply:
This one is reference in Jenkinsfile in ACA but not present here hence why added

Reply:
I would suggest to use env var names containing the first names of the automation users, as otherwise it will be difficult to always match which password is used for which user.

Reply:
It's not how we named them before for Becky, Benjamin and Bill, but better improve the naming now then later.
We can also consider renaming other env vars if it's not a big deal. With the growing amount of automation data, naming conventions and other similar stuff is becoming increasingly important.

Reply:
I agree: particularly in light of the 'Actor Review' that is ongoing.  However I feel it is best resolved in a separate tech debt ticket when we can address the existing accounts 'Becky, Benjamin, Bill, Emma' across all repositories that use them.

Reply:
Have been using this PR to test locally and all works ok, so happy to merge as is? Agree with Matt regarding separate technical debt ticket relating to the current ongoing work here

https://tools.hmcts.net/confluence/display/RCCD/MCA+BEFTA+Actors+list